### PR TITLE
feat: add Muziekschatten - onderwerpen

### PR DESCRIPTION
**Pull request by EnnoMeijers**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,2 @@
-build
```
Comment (ddeboer) [inline .gitignore:1]: Just so I understand, why did you remove this? The `build` dir should not be checked in.

Comment (EnnoMeijers) [reply .gitignore:1]: This was necessary in order to run the api against a dev branch of the catalog (included by pointing to dev branch in the github repo). Since this requires an install and compile it also needs the build files included in node_modules. Please let me know if there is way of doing this. Maybe we should remove the build directory again before merging into master.

File: catalog/datasets/muziekschatten-onderwerpen.jsonld [hunk 2]
```diff
@@ -0,0 +1,34 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "http://data.muziekschatten.nl/onderwerpen",
+  "@type": "Dataset",
+  "name": [
+    {
+      "@language": "nl",
+      "@value": "Muziekschatten - onderwerpen"
```
Comment (ddeboer) [inline catalog/datasets/muziekschatten-onderwerpen.jsonld:8]: For existing sources, we have `name colon subject`, e.g. `Muziek: genres en stijlen`. Could we standardise on that notation? This one would then become `Muziekschatten: onderwerpen`.

File: catalog/datasets/muziekschatten-onderwerpen.jsonld [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    }
+  ],
+  "alternateName": [
+    {
+      "@language": "nl",
+      "@value": "SOM"
```
Comment (ddeboer) [inline catalog/datasets/muziekschatten-onderwerpen.jsonld:14]: Is the source Muziekschatten: onderwerpen indeed abbreviated as SOM? Or would that only be the alternateName of the creator, i.e. Stichting Omroep Muziekbibliotheek. Please note that you can leave out the alternateName if you want to. E.g. RKDArtists is just RKDArtists, without any abbreviation.

File: .gitignore (hunk 1 above)
Comment (ddeboer) [reply .gitignore:1]: Removed in ae5a5492af009ff0687ccc0645c714d1caff512c.

Comment (ddeboer) [reply .gitignore:1]: I explained local development in 249d4d12b2efcd9291f64cba337d66a910ceb41b. Please let me know if that clears up things.

File: catalog/datasets/muziekschatten-onderwerpen.jsonld (hunk 3 above)
Comment (EnnoMeijers) [reply catalog/datasets/muziekschatten-onderwerpen.jsonld:14]: The precise relation between Muziekschatten and SOM is not completely clear to me. It looks like the SOM data is a subcollection of Muziekschatten, see also http://www.muziekbibliotheekvandeomroep.nl/. I left the SOM value in here in order to make a reference this and to the dataset in the previous version op the prototype. In that version the source is called Stichting Omroep Muziekbibliotheek and it points to the muziekschatten endpoint. As far as I know SOM doesn't exists any more and it's collection is part of the Muziekschatten service.

File: catalog/datasets/muziekschatten-onderwerpen.jsonld (hunk 2 above)
Comment (EnnoMeijers) [reply catalog/datasets/muziekschatten-onderwerpen.jsonld:8]: Good suggestion, I'll change this.

File: catalog/datasets/muziekschatten-onderwerpen.jsonld [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "http://data.muziekschatten.nl/onderwerpen",
```
Comment (ddeboer) [inline catalog/datasets/muziekschatten-onderwerpen.jsonld:3]: This is made up, isn’t it? Perhaps https://data.muziekschatten.nl/#onderwerpen is better because at least that resolves to some URL.

File: catalog/datasets/muziekschatten-onderwerpen.jsonld (hunk 3 above)
Comment (ddeboer) [reply catalog/datasets/muziekschatten-onderwerpen.jsonld:14]: Interesting! Creator https://www.muziekschatten.nl is indeed called Stichting Omroep Muziek (SOM). The [source itself](http://data.muziekschatten.nl/som/) seems to be called Stichting Omroep Muziek Bladmuziekcatalogus. Are we selecting only subjects (‘onderwerpen’) from that source? Shouldn’t we also select artists and songs?

@sdevalk You previously worked with them. Do you have a suggestion how to name both this source its creator?

Comment (sdevalk) [reply catalog/datasets/muziekschatten-onderwerpen.jsonld:14]: > @sdevalk You previously worked with them. Do you have a suggestion how to name both this source its creator?

I'll ask SOM!

Comment (sdevalk) [reply catalog/datasets/muziekschatten-onderwerpen.jsonld:14]: I had contact with SOM:

1. The `name` of the organization/publisher should be `Stichting Omroep Muziek`, the `alternateName` should be `SOM` and the `@id` should be `https://www.omroepmuziek.nl/`

2. The `name` of the dataset should be/contain `Muziekschatten`. Or in case of the 'onderwerpen': `Muziekschatten: onderwerpen`.

Comment (EnnoMeijers) [reply catalog/datasets/muziekschatten-onderwerpen.jsonld:14]: Changed the definitions as suggested above.
